platform/install/advanced/air-gapped.mdx

```diff
@@ -2,36 +2,41 @@
 title: Air-Gapped Install
 sidebar_label: Air-Gapped
 sidebar_position: 1
+description: Learn how to install vCluster Platform in air-gapped, offline, or VPC environments with restricted network connectivity.
 ---
 
 import Flow, { Step } from "@site/src/components/Flow";
 import PartialAdminSetVersion from "../../_partials/install/set-version.mdx";
 import PartialAdminUpgrade from "../../_partials/install/upgrade.mdx";
+import Mark from "@site/src/components/Mark";
 
+<!-- vale off -->
 ## Air-Gapped / Offline / VPC Clusters
+<!-- vale on -->
+If your cluster is air-gapped, within a VPC, or has restricted network connectivity, you may encounter issues validating the platform license and pulling required container images.
 
-If your cluster is air-gapped, within a VPC or restricts network connectivity in other ways, you
-may run into issues validating the vCluster Platform license and pulling container images required for vCluster Platform.
-
-To solve the licensing issue, you can either use allow HTTP requests from the
-vCluster Platform pods to
+To solve the licensing issue, you can either use allow HTTP requests from
+the platform pods to
 `https://admin.loft.sh/*` (this is the preferred option), or use an
 [offline license key](#offline-license-key).
 
-To solve not being able to pull images from Docker Hub, you can
-[use a private image registry](#offline-images) to host your own copies of the vCluster Platform images that
-your cluster can reach.
+To address image pulling issues from Docker Hub, you can [use a private image
+registry](#offline-images) to host your own copies of the platform images
+accessible to your cluster.
 
 ### Prerequisites
 
 - A running air-gapped Kubernetes cluster `v1.26.x`, `v1.27.x`, or `v1.28.x`
```

Comment:
These versions are out of date. I'm wondering how we should handle this as a whole.

Reply:
Capturing for later: https://linear.app/loft/issue/DOC-301/update-kubernetes-versions-requirements-for-air-gapped